e2e/resources/python-code/import-libs.py

```diff
@@ -7,7 +7,7 @@
 import numpy
 import os
 import pandas
-# TODO(RW-8293): Currently breaks plotting, import again once fixed.
+# TODO(RW-8293): Breaks plotting on 2.0.14, import again once fixed.
```

Comment:
Going ot keep the test change in so this doesn't cause failures on cached workspaces.